<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* macOS Separator Fix */

QMainWindow::separator {
    background: transparent;
    width: 0px;
    height: 0px;
}

QMainWindow {
	border: 0px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="ButtonNavigation" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="RoundButton" name="OnButton">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/Resources.qrc">
           <normaloff>:/buttons/BlackPlay.svg</normaloff>
           <normalon>:/buttons/BlackPause.svg</normalon>:/buttons/BlackPlay.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="RoundButton" name="radioButton">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/Resources.qrc">
           <normaloff>:/buttons/BlackNote.svg</normaloff>
           <normalon>:/buttons/BlueNote.svg</normalon>:/buttons/BlackNote.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="RoundButton" name="mapButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/Resources.qrc">
           <normaloff>:/buttons/BlackNavigation.svg</normaloff>
           <normalon>:/buttons/BlueNavigation.svg</normalon>:/buttons/BlackNavigation.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="RoundButton" name="settingsButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/Resources.qrc">
           <normaloff>:/buttons/BlackSettings.svg</normaloff>
           <normalon>:/buttons/BlueSettings.svg</normalon>:/buttons/BlackSettings.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="Navigation">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="SettingsPage" name="Settings"/>
      <widget class="RadioPage" name="RadioDash"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoundButton</class>
   <extends>QPushButton</extends>
   <header>widgets/RoundButton.h</header>
  </customwidget>
  <customwidget>
   <class>RadioPage</class>
   <extends>QWidget</extends>
   <header>pages/RadioPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsPage</class>
   <extends>QWidget</extends>
   <header>pages/SettingsPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
